Misc/NEWS.d/next/C_API/2025-04-13-20-52-39.gh-issue-132470.UqBQjN.rst

```diff
@@ -0,0 +1,14 @@
+Creating a ctypes.CField with a byte_size that does not match the actual
+type size now raises a ValueError instead of crashing the interpreter. #
+Please enter the relevant GitHub issue number here: #
+
+# # Uncomment one of these "section:" lines to specify which section # this
+entry should go in in Misc/NEWS.d. # #.. section: Security #.. section: Core
+and Builtins #.. section: Library #.. section: Documentation #.. section:
+Tests #.. section: Build #.. section: Windows #.. section: macOS #..
+section: IDLE #.. section: Tools/Demos #.. section: C API
+
+# Write your Misc/NEWS.d entry below.  It should be a simple ReST paragraph.
+# Don't start with "- Issue #<n>: " or "- gh-issue-<n>: " or that sort of
+stuff.
+###########################################################################
```

Comment:
```suggestion
Creating a :class:`ctypes.CField` with a *byte_size* that does not match the actual
type size now raises a :exc:`ValueError` instead of crashing the interpreter.
```